lib/data/legalisation_documents_data.yml

```diff
@@ -178,7 +178,7 @@ letter-of-invitation:
 letter-of-no-trace:
   heading: Letter of no trace
   type: letter of no trace
-  group: birth_death
+  group: letter_no_trace
```

Comment:
@ikennaokpala: Minor: Ideally this change would've been made in the previous commit.